### PR TITLE
Updated readme for binary functions

### PR DESCRIPTION
**Pull request by leesafini**
AnalyticalGraphicsInc/cesium#4688

@lilleyse Updated the Styling readme to include the binary functions. Not sure if there are better arguments to be using over angle and temperature, or if there might be more clarifying examples.

### CODE REVIEW

File: Styling/README.md [hunk 1]
```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
 }
 ```
 
+#### atan2
+
+`atan2(value1 : Number, value2 : Number) : Number`
```
Comment (lilleyse) [inline Styling/README.md]: value1 -> y, value2 -> x is common terminology.

File: Styling/README.md [hunk 2]
```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
...
+
+Returns the angle `theta` in polar coordinates `(r, theta)` for a point with Cartesian coordinates `(value1, value2)`.
```
Comment (lilleyse) [inline Styling/README.md]: Maybe there is a simpler description? MDN defines it as:

> The Math.atan2() function returns the arctangent of the quotient of its arguments.

Comment (lilleyse) [inline Styling/README.md]: Also like the other math functions make sure to mention the result is in radians.

File: Styling/README.md [hunk 3]
```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
...
+
+#### pow
+
+`pow(value1 : Number, value2 : Number) : Number`
+
+Returns `value1` raised to the power of `value2`.
+
+```json
+{
+    "show" : "pow(${Angle}, ${temperature}) > 1.0"
```
Comment (lilleyse) [inline Styling/README.md]: Instead of `Angle`, maybe something like `Density`. For consistency upper-case `temperature`.

File: Styling/README.md [hunk 4]
```diff
@@ -626,7 +630,7 @@ Converts the number from radians to degrees.
 
 `sqrt(value : Number) : Number`
 
-Returns the square root of value if value >= 0. Returns NaN when value < 0.
+Returns the square root of `value` if `value >= 0`. Returns NaN when `value < 0`.
```
Comment (lilleyse) [inline Styling/README.md]: Maybe as well also put quotes around `NaN`

File: Styling/README.md (hunk 3 above)
Comment (lilleyse) [inline Styling/README.md]: Also use the uppercases in the sample code sections below.

File: Styling/README.md [hunk 5]
```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
...
+
+#### min
+
+`min(value1 : Number, value2 : Number) : Number`
+
+Returns `value1` if `value1 < value 2` and `value2` if `value2 < value1`.
+
+```json
+{
+    "show" : "min(${Angle}, ${temperature}) > 0.0"
```
Comment (lilleyse) [inline Styling/README.md]: Other variable names are better for min and max, but I don't have any suggestions.

File: Styling/README.md [hunk 6]
```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
...
+{
+    "show" : "pow(${Angle}, ${temperature}) > 1.0"
+}
+```
+
+#### min
+
+`min(value1 : Number, value2 : Number) : Number`
+
+Returns `value1` if `value1 < value 2` and `value2` if `value2 < value1`.
```
Comment (lilleyse) [inline Styling/README.md]: The descriptions for min/max on http://www.shaderific.com/glsl-functions/ are easier to follow.

File: Styling/README.md [hunk 7]
```diff
@@ -513,6 +513,10 @@ The following built-in functions are supported by the styling language:
 * [`atan`](#atan)
 * [`radians`](#radians)
 * [`degrees`](#degrees)
+* [`atan2`](#atan2)
```
Comment (lilleyse) [inline Styling/README.md]: Put `atan2` after `atan`, even though its not unary - here and below.

File: Styling/README.md (hunk 5 above)
Comment (lilleyse) [inline Styling/README.md]: Width and Height could be good

**Comment by leesafini** [comment]
@lilleyse How does this look?

File: Styling/README.md [hunk 8]
```diff
@@ -598,6 +620,18 @@ Returns the arctangent value of the number (in radians).
...
 
+#### atan2
+
+`atan2(y : Number, x : Number) : Number`
+
+Returns the arctangent of the quotient of `y` and `x` (in radians).
+
+```json
+{
+    "show" : "atan2(${Angle}, ${temperature}) > 0.0"
```
Comment (lilleyse) [inline Styling/README.md]: I would use different variable names here. `GridY` and `GridX` might be good names.

File: Styling/README.md [hunk 9]
```diff
@@ -622,21 +656,39 @@ Converts the number from radians to degrees.
...
+
+#### min
+
+`min(value1 : Number, value2 : Number) : Number`
+
+Returns the smaller of the two arguments.
+
+```json
+{
+    "show" : "min(${Width}, ${Height}) > 0.0"
```
Comment (lilleyse) [inline Styling/README.md]: For this and max, use a more realistic value like `> 10.0`.

**Comment by lilleyse** [comment]
Looks great. Only these small comments.

**Comment by leesafini** [comment]
@lilleyse updated!

File: Styling/README.md [hunk 10]
```diff
@@ -598,6 +620,18 @@ Returns the arctangent value of the number (in radians).
 }
 ```
 
+#### atan2
+
+`atan2(GridY : Number, GridX : Number) : Number`
+
+Returns the arctangent of the quotient of `GridY` and `GridX` (in radians).
```
Comment (lilleyse) [inline Styling/README.md]: For this one I meant to rename `Angle` and `temperature` to `GridY` and `GridX`, but the function definition/description can stay `y` `x`.

**Comment by leesafini** [comment]
Hahaha sorry, wasn't sure which one you meant. Changed now.

**Comment by lilleyse** [comment]
Thanks, all good now.

**Pull request merged**